internal/provider/resource_cloudflare_pages_project.go

```diff
@@ -170,7 +175,7 @@ func resourceCloudflarePagesProjectRead(ctx context.Context, d *schema.ResourceD
 		return diag.FromErr(fmt.Errorf("error reading cloudflare pages project %q: %w", projectName, err))
 	}
 
-	d.SetId(res.ID)
```

Comment:
what's the thoughts here? where possible, we want to use IDs since names are not unique.